packages/expo-media-library/ios/MediaLibraryUtilities.swift

```diff
@@ -55,7 +55,7 @@ func stringifyAlbumType(type: PHAssetCollectionType) -> String {
 
 func exportAssetInfo(asset: PHAsset) -> [String: Any?]? {
   if var assetDict = exportAsset(asset: asset) {
-    assetDict["location"] = exportLocation(location: asset.location)
+    assetDict["location"] = exportLocation(location: asset.location) ?? NSNull()
```

Comment:
[this assertion was failing](https://github.com/expo/expo/blob/8b857d8e6741696ed74a783d86454d616d17396b/apps/test-suite/tests/MediaLibrary.js#L197) - the `location` entry wasn't present in the response. now it's `null`.

Reply:
I think it's a good idea, this is a typical use case for the `Record`